packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -964,3 +964,24 @@ def _get_prev_owner(cls, linked_list: List[str], target: str) -> str:
             # if target is the first element in the list, the root is pointing to it
             return root
         return linked_list[index]
+
+    @classmethod
+    def get_owners(
+        cls,
+        ledger_api: EthereumApi,
+        contract_address: str,
+    ) -> JSONLike:
+        """
+        Get the safe owners.
+
+        :param ledger_api: the ledger API object
+        :param contract_address: the contract address
+        :return: the safe owners
+        """
+        ledger_api = cast(EthereumApi, ledger_api)
+        safe_contract = cls.get_instance(ledger_api, contract_address)
+        owners = [
+            ledger_api.api.toChecksumAddress(owner)
+            for owner in safe_contract.functions.getOwners().call()
+        ]
```

Comment:
Just a reminder, we have `contract_method_call` available. Using this would allow more uniformity across our repos.

```suggestion
        owners = ledger_api.contract_method_call(
            contract_instance=safe_contract,
            method_name="getOwners",
        )
        owners = [
            ledger_api.api.toChecksumAddress(owner)
            for owner in owners
        ]
```

Reply:
It caused some problems #1497